src/components/Button/Button.js

```diff
@@ -39,12 +39,12 @@ const styles = StyleSheet.create({
     alignItems: 'center'
   },
   buttonText: {
-    color: colors.white,
+    color: COLORS.white,
     fontSize: 18,
     fontFamily: fonts.inter[600]
   },
   disabledbutton: {
-    backgroundColor: colors.gray1,
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color for the button and the background color for the disabled button have been updated to use `COLORS.white` and `COLORS.lightgrey` respectively from the `theme` module instead of `colors.white` and `colors.gray1` from the `colors` module.
```diff
-  color: colors.white,
+  color: COLORS.white,
-  backgroundColor: colors.gray1,
+  backgroundColor: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->